src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/PageAction.cs

```diff
@@ -28,6 +28,10 @@ public PageAction(IWebDriver driver, ITestOutputHelper helper)
             // Initialize the page objects
             pages = new PageObjects().Pages;
         }
-        
+        public void GoBackOnePage()
```

Comment:
I think this should go in the Common class (it's likely to be used in a lot of places

Reply:
Good point, done.